.gitignore

```diff
@@ -26,3 +27,5 @@ tools/smartdnsblock/bin/*
 !tools/smartdnsblock/bin/*.exe
 universal.apk
 xcuserdata/
+do_install_script.ts
```

Comment:
Is this in the root?

I'd rather keep the .gitignore localized in the respective folder.

Reply:
Please create a .gitignore in the directory where do_install_script.ts is created, so it's localized there.
I'd say same for `/server_manager` to ignore the node_modules.

It's easier to manage these if they are in context.